templates/ironicinspector/config/httpd.conf

```diff
@@ -30,8 +30,12 @@ CustomLog /dev/stdout proxy env=forwarded
   ServerName {{ $vhost.ServerName }}
 
   ## Request header rules
-  ## as per http://httpd.apache.org/docs/2.2/mod/mod_headers.html#requestheader
-  RequestHeader set X-Forwarded-Proto "https"
+  ## as per http://httpd.apache.org/docs/2.4/mod/mod_headers.html#requestheader
+  {{- if $vhost.TLS }}
+    RequestHeader setIfEmpty X-Forwarded-Proto "https"
```

Comment:
Let's change the indentation here, to same level as `ServerName`?
The rendered results will be "prettier".
ie. 
```
<VirtualHost *:5050>
  ServerName {{ $vhost.ServerName }}

  ## Request header rules
  ## as per http://httpd.apache.org/docs/2.4/mod/mod_headers.html#requestheader
  RequestHeader setIfEmpty X-Forwarded-Proto "https"

  ... snip ...
</VirtualHost>
```